cmd/fabric-ca-server/servercmd.go

```diff
@@ -154,6 +154,8 @@ func (s *ServerCmd) registerFlags() {
 	pflags.StringVarP(&s.homeDirectory, "home", "H", "", fmt.Sprintf("Server's home directory (default \"%s\")", filepath.Dir(cfg)))
 	util.FlagString(s.myViper, pflags, "boot", "b", "",
 		"The user:pass for bootstrap admin which is required to build default config file")
+	util.FlagString(s.myViper, pflags, "bootfile", "f", "",
```

Comment:
Usage question here: when I try to specify a username and passfile, it complains because I haven't specified a password after a colon separator:
`fabric-ca-server init -b admin -f pass.file`
`Error: Failed to create default configuration file: The value 'admin' on the command line is missing a colon separator`

While it isn't too difficult to add the `: ` after `admin`, it feels like we shouldn't require that when `-f` is used. Any reason that shouldn't be the case?

Reply:
As we discussed, this was a design choice (trying to change as little as possible). The `init` and `start` commands still require both `-b admin:pass` with the `-f pass.file` flag specified. The _pass_ from `-b user:pass` will be used as a backup when a password cannot be read from _pass.file_ when starting server. 

Another option as you suggested is to allow only `-b admin` when `-f pass.file` is provided then
1.  Allow the server to error on startup if the _pass.file_ cannot be read. 
2. Error when creating the default config file if the _pass.file_ doesn't exist, cannot be read, etc.

Reply:
@sykesm Any thoughts / opinions here?